applications/CompressiblePotentialFlowApplication/tests/cpp_tests/test_transonic_perturbation_potential_element.cpp

```diff
@@ -156,6 +234,45 @@ KRATOS_TEST_CASE_IN_SUITE(PingTransonicPerturbationPotentialFlowElementLHS, Comp
     }
 }
 
+KRATOS_TEST_CASE_IN_SUITE(TransonicPerturbationPotentialFlowElementEquationId, CompressiblePotentialApplicationFastSuite) {
+    Model this_model;
+    ModelPart& model_part = this_model.CreateModelPart("Main", 3);
+
+    GenerateTransonicPerturbationElement(model_part);
+    GenerateTransonicPerturbationUpwindElement(model_part);
+
+    Element::Pointer pCurrentElement = model_part.pGetElement(1);
+    pCurrentElement->Initialize(model_part.GetProcessInfo());
+
+    for (unsigned int i = 0; i < 3; i++) {
+        pCurrentElement->GetGeometry()[i].AddDof(VELOCITY_POTENTIAL);
+    }
+
+    Element::DofsVectorType CurrentElementalDofList;
+    pCurrentElement->GetDofList(CurrentElementalDofList, model_part.GetProcessInfo());
+
+    for (int i = 0; i < 3; i++) {
+        CurrentElementalDofList[i]->SetEquationId(i);
+    }
```

Comment:
I would do this for some random numbers to make sure it works:

```suggestion
    std::vector<int> ids{23, 74, 55};
    for (int i = 0; i < 3; i++) {
        CurrentElementalDofList[i]->SetEquationId(ids[i]);
    }
```